.gitignore

```diff
@@ -139,3 +139,6 @@ timing_stats*
 sbi-logs
 matts_tests
 .vscode
+
+cmass/conf/nbody/pin_1gpch*.yaml
```

Comment:
Is this still necessary, or only for testing?

Reply:
This has now been removed